### ISSUE TITLE
'Zombie' tooltips in tree-map

### ISSUE THREAD

**Issue by m-ibm** [alexandersimoes/d3plus#493]
If navigating away from a page with a tree-map while a tooltip is displayed, the tooltip is not hidden/destroyed. We're using version 1.9.2 (not 100% sure as plugin does not contain version number).

See screen shot for an example:

![d3plus_tooltip_bug](https://cloud.githubusercontent.com/assets/13468737/16118584/727d2986-33e0-11e6-8966-7ff2d87e40cd.png)

Refreshing the page makes it disappear, however, this also re-sends the requests to server which may perform calculations (luckily just get, no post).

This happens pretty consistently when navigation away through a modal that's displayed on top of the tree-map. Modal closes, tooltip appears just as new page starts loading.

Tree-map should probably catch navigation away from pages event, or maybe something more generic as there might be other cases when this happens.

**Comment by davelandry** [comment]
please update to the latest version first, as we've done some work with tooltips since then.

you can see what version number you are using by typing this into the browser's development console:
```js
d3plus.version
```

additionally, which browsers and OSes are you seeing this bug in?

**Comment by m-ibm** [comment]
Yes, this happens with the latest version as well. My current version is "1.9.3 - Cornflower"
I see this on Windows 7 (haven't tried Linux) and Chrome. It doesn't seem to happen on Firefox.

I do get new problems with 1.9.5 :-(
